docs/en/writing-running-appium/web/hybrid.md

```diff
@@ -82,9 +82,12 @@ driver.quit();
 ```
 
 ```ruby
-# ruby
+# ruby_lib_core
 # assuming we have a set of capabilities
-@driver = Selenium::WebDriver.for(:remote, :desired_capabilities => capabilities, :url => SERVER_URL)
+@driver = Appium::Core.for(url: SERVER_URL, desired_capabilities: capabilities).start_driver
+# ruby_lib
+# opts = { caps: capabilities, appium_lib: { custom_url: SERVER_URL }}
+# @driver = Appium::Driver.new(opts, true).start_driver
```

Comment:
Why are these two commented out?

Reply:
Without comment-out, the code means overriding `@driver`, but I thought it would be better to put two cases. thus, i choosen commenting out the later method.